data/json/items/comestibles/fruit_dishes.json

```diff
@@ -269,5 +269,29 @@
     "stack_size": 24,
     "fun": 3,
     "vitamins": [ [ "vitC", 3 ], [ "calcium", 2 ] ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "fruit_alcohol_preserved",
+    "name": "fruit in alcohol",
```

Comment:
"fruit in alcohol" sounds sort of awkward in my opinion - maybe "brandied fruit"? What do you think?

Reply:
Perfect! Thanks.